### DISCUSSION TITLE
About \n in table

### DISCUSSION THREAD

**Discussion by ru88s** [supabase/supabase#1568] [Questions] [answered]
There is a \n in the text that disappears when I select it.
Is this a bug?

https://gyazo.com/4e9b5d521cf6f6c2025da0f90e0f9c4a

**Comment by silentworks** [comment, marked as answer]
It might be, how did you insert this data? was it through the supabase-js library or inside of the Supabase dashboard UI?

**Reply by ru88s** [reply]
I used  react-hook-form.
Form line breaks.
https://react-hook-form.com/
